robyn/__init__.py

```diff
@@ -162,16 +163,19 @@ def init_processpool(socket):
 
             process_pool = init_processpool(socket)
 
-            logger.info(f"{Colors.OKGREEN}Press Ctrl + C to stop \n{Colors.ENDC}")
-            try:
-                for process in process_pool:
-                    process.join()
-            except KeyboardInterrupt:
+            def terminating_signal_handler(_sig, _frame):
                 logger.info(
                     f"{Colors.BOLD}{Colors.OKGREEN} Terminating server!! {Colors.ENDC}"
                 )
                 for process in process_pool:
                     process.kill()
+
+            signal.signal(signal.SIGINT, terminating_signal_handler)
+            signal.signal(signal.SIGTERM, terminating_signal_handler)
```

Comment:
https://github.com/sansyrox/robyn/issues/317

This might help us with this. @AntoineRR , do you have a windows machine by any chance?

Reply:
I dual boot Windows and Linux currently yes. I tried it on Windows and my code didn't help with this issue unfortunately... I might try to fix it later, doesn't seem easy tho 🤔 It seems to be linked with spawning subprocesses in Windows (there is an if statement in the code currently that changes the behaviour on Windows to not use `Process`)